components/molecule/selectField/src/index.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeField from '@s-ui/react-molecule-field'
+import MoleculeSelect from '@s-ui/react-molecule-select'
+
+const getErrorState = (success, error) => {
+  if (error) return true
+  if (success) return false
+}
+
+class MoleculeSelectField extends Component {
+  refSelect = React.createRef()
+  id = this.props.id || `MoleculeSelect-${+new Date()}`
+
+  handleClick = () => {
+    const {current: domSelect} = this.refSelect
+    if (domSelect) domSelect.focus()
+  }
+
+  render() {
+    const {id, refSelect, handleClick} = this
+    const {
+      label,
+      successText,
+      errorText,
+      helpText,
+      inline,
+      children, // eslint-disable-line
+      ...props
+    } = this.props
+    const errorState = getErrorState(successText, errorText)
```

Comment:
I had to go to the specification of the `function` to know what this does.

Due to the function returns `true` or `false` depending on if it the components has an errors text, what do you think about naming it `hasErrors` or `hasErrorMessage`?

Reply:
You're right. `hasErrors` it is